### DISCUSSION TITLE
DisabledDates not working

### DISCUSSION THREAD

**Discussion by algomera** [t1m0n/air-datepicker#601] [Q&A] [answered]
Hello everyone,
after updating the package to v3.5.0 I tried to insert the "DisabledDates" property inside the calendar instance.
I have tried entering the required Set by entering the dates in various formats but never succeeded....

Can someone kindly explain or show me how to enter the dates to be disabled?

I am trying to create a Blade/Alpine component that will interface with Livewire. Everything works fine except for this feature which is essential for what I am doing.

Thank you very much to anyone who can help me!

**Comment by t1m0n** [comment, marked as answer]
@algomera Hi. `disabledDates` is a property of AirDatepicker instance, it's not an option. If you would like to have some dates disabled on start, you can use `onRenderCell` option and return `{disabled: true}` when you want this or that date to be disabled.

Another way to disabled dates is using `disabledDate` method:

```javascript
const dp = new AirDatepicker('el');

dp.disableDate('2024-03-23')
// or passing array if you need multiple dates to be disabled
dp.disableDate(['2024-03-23', '2024-03-24'])

// After you can access all disabled dates like this
console.log(dp.disabledDates)

```

**Reply by algomera** [reply]
Thank you very much for the answer and the example..
I resolve it by passing an array of dates with `dp.disableDate()` method!

Have a nice day!

**Comment by algomera** [comment]
Hi, sorry to reopen the conversation but it is a question related to the problem in the thread.
Is there a possibility to disable date selection (range) if there is a disabled date within the range via the "disableDate()" method?

Thank you
